### PR TITLE
secure/fix a few URLs

### PR DESCRIPTION
**Pull request by vszakats**
Detected via https-everywhere rules.

### CODE REVIEW

**Comment by vszakats** [comment]
Something else to be done with this PR to get it merged?

File: src/icu4c.mk [hunk 1]
```diff
@@ -13,7 +13,7 @@ $(PKG)_URL      := https://ssl.icu-project.org/files/$(PKG)/$($(PKG)_VERSION)/$(
 $(PKG)_DEPS     := cc
 
 define $(PKG)_UPDATE
-    $(WGET) -q -O- 'http://bugs.icu-project.org/trac/browser/icu/tags' | \
+    $(WGET) -q -O- 'https://ssl.icu-project.org/trac/browser/icu/tags' | \
```
Comment (jonaski) [inline src/icu4c.mk]: This site doesn't work for me with https, I get connection refused.

Comment (vszakats) [reply src/icu4c.mk]: Indeed, thanks! — Their SSL support is not operating, even though it has been for many years.
I've undone this update.

**Comment by vszakats** [comment]
With the icu4c link update undone, is there some remaining concern?

**Comment by tonytheodore** [comment]
Thanks!

Note `make cleanup-style` will fix tab issues - there's also one in curl.mk but I don't have a machine I can push from at the moment.

**Pull request merged**